packages/back-end/src/app.ts

```diff
@@ -469,6 +474,22 @@ app.post(
   "/experiments/:id/visual-changeset",
   experimentsController.postVisualChangeset
 );
+app.post(
+  "/experiments/launch-checklist",
+  experimentLaunchChecklistController.postExperimentLaunchChecklist
+);
+app.put(
+  "/experiments/launch-checklist/:id",
+  experimentLaunchChecklistController.putExperimentLaunchChecklist
+);
+app.get(
+  "/experiments/launch-checklist",
+  experimentLaunchChecklistController.getExperimentCheckListByOrg
+);
+app.put(
+  "/experiments/:id/launch-checklist",
```

Comment:
This endpoint is for a specific experiment, so make it singular `/experiment/:id` instead of `/experiments/:id`

Reply:
Updated in [this commit](https://github.com/growthbook/growthbook/pull/1778/commits/0f8c23281d3a1995009bffe6cc2968ac1de270a1).